### DISCUSSION TITLE
Is there a solution in place for changing TV's "picture mode"?

### DISCUSSION THREAD

**Discussion by rorosaurus** [awawa-dev/HyperHDR#707] [Q&A] [answered]
Depending on the time of day, sunlight, etc. we rotate between a few different picture modes for our TV. Sometimes we want a brighter, cooler image, and sometimes we want a dimmer, warmer image.

Is there a solution in place for handling this change? Now that I think of it, how do you calibrate the colors to match your TV's in the first place?

I assume the answer is: control it over the web or use the JSON API for remote control and/or find a way to automate it with HomeAssistant?

**Comment by awawa-dev** [comment, marked as answer]
> Is there a solution in place for handling this change? Now that I think of it, how do you calibrate the colors to match your TV's in the first place?

There is a calibration LUT tool to perform automatic calibration as there are some differences how grabber encode YUV colors (MJPEG codec also uses YUV internally)

> I assume the answer is: control it over the web or use the JSON API for remote control and/or find a way to automate it with HomeAssistant?

If you're using it on Linux, e.g. RPi, why not just use a cron locally that will invoke a simple bash script that invokes JSON API commands (via curl)? You have an editor for creating JSON commands built into HyperHDR (Advanced->JSON API tab)

**Reply by awawa-dev** [reply]
MQTT can also be used
